### DISCUSSION TITLE
Has purgeCSS as an option for configuration - v3+?

### DISCUSSION THREAD

**Discussion by weilinzung** [tailwindlabs/tailwindcss#6704] [General]
I noticed v3+ has removed purgeCSS, but I think for us it is useful since we want to build landing websites and always want them to load fast as possible.

Would be possible to bring it back but as an option? and turn off as default?

Or
Provide a doc on how to add purgeCSS manually with PostCss?

Thanks!

**Comment by alexanderbluhm** [comment]
Hi @weilinzung 
You don't have to worry about loading time in v3. The new Just-In-Time compiler is used as default and you have minimal css size in both development and production.
You can read more about the JIT Compiler [here](https://tailwindcss.com/blog/just-in-time-the-next-generation-of-tailwind-css)
